launcher/Makefile

```diff
@@ -28,7 +28,7 @@ $(TARGET).vpk: $(TARGET).velf
 	cp -f param.sfo ../release/sce_sys/param.sfo
 	
 	#------------ Comment this if you don't have 7zip ------------------
-	7z a -tzip ./$(TARGET).vpk -r ..\release\sce_sys\* ..\release\eboot10.bin ..\release\eboot22.bin ..\release\eboot23.bin ..\release\eboot20.bin ..\release\eboot30.bin ..\release\eboot00.bin ..\release\eboot.bin 
+	7z a -tzip ./$(TARGET).vpk -r ..\release\*
```

Comment:
Are you sure this works properly creating subdirectories?

Reply:
Yes I tested it and it works.